### DISCUSSION TITLE
Is there any licensing issue with iText Jar being used in Birt Runtime?

### DISCUSSION THREAD

**Discussion by utkarsh-ut** [eclipse-birt/birt#919] [Q&A] [answered]
Want to understand that Is there any licensing issue with iText Jar being used in Birt Runtime?

**Comment by claesrosell** [comment, marked as answer]
The version of iText used in Birt versions before 4.9 is licensed under MPL/LGPL.
Birt 4.9.0 is using OpenPDF which is also licensed under MPL/LGPL.

Does this answer your question?

**Reply by utkarsh-ut** [reply]
Ther Birt we are using internally use the com.lowagie.text_2.1.7.v201004222200.jar, Do we need to pay anything to anybody for commercial use?

**Reply by claesrosell** [reply]
Sorry, I will not give you any legal advice

**Reply by utkarsh-ut** [reply]
Ok Can you tell me a bit about MPL/LGPL, I a bit new for this.

**Reply by claesrosell** [reply]
There is no short answer to this question and far wiser people then me has tried to explain the implications of various copyleft licenses. I encourage you to do some deep digging on the internet.
